chemie/shop/templates/shop/happy-hour.html

```diff
@@ -38,16 +38,15 @@
 {% endblock breadcrumbs %}
 
 {% block content %}
-<form method="POST"> {% csrf_token %}
-  <table>
+<form method="POST" id="happyhourForm"> {% csrf_token %}
     {% crispy form %}
-  </table>
+    <input id="submit_value" class="btn btn-secondary" type="button" name="submit_value" value="Aktiver happy hour" style="color:black; margin: 1rem; background-image: url('/static/images/fireworks_background.png'); background-color: transparent; background-size: cover" href="{% url "shop:activate-happyhour" %}">
 </form>
 {% endblock content %}
 
 {% block footer_script %}
 <script>
-  document.getElementById("activate-button").addEventListener("click", function (event) {
+  document.getElementById("submit_value").addEventListener("click", function (event) {
```

Comment:
Maybe name this something more explanatory than submit_value. For instance form-submit-button?